perl-package/test.sh

```diff
@@ -0,0 +1,16 @@
+MXNET_HOME=$1
```

Comment:
the HOME is the perl workspace where perl will install its intermediate libs for the tests.
The /workspace as the place for mxnet/lib dir I determined because of  http://ci.mxnet.io/blue/organizations/jenkins/mxnet/detail/PR-7170/3/pipeline/230
I see multiple messages like this
```
/workspace/python/mxnet/module/base_module.py
```
Which suggested to me that the /workspace/ dir is where the mxnet home is